packages/website/public/schema.yml

```diff
@@ -416,6 +425,34 @@ components:
         format: int32
         minimum: 1
         maximum: 1000
-        default: 25
+        default: 10
+    offset:
+      description: Specifies the offset of returned results, used for pagination.
+      name: offset
+      in: query
+      required: false
+      schema:
+        type: integer
+        format: int32
+        minimum: 0
+        default: 0
+    sortBy:
+      description: Sorts results by a given parameter.
+      name: sortBy
+      in: query
+      required: false
+      default: Name
+      schema:
+        type: string
+        enum: [Date, Name]
+    sortOrder:
+      description: Sorts the order of the results.
+      name: sortOrder
+      in: query
+      required: false
+      default: Desc
+      schema:
+        type: string
+        enum: [Asc, Desc]
```

Comment:
Decided to document this previously hidden functionality as it confused me when initially looking. Is this ok? Or are we hiding `sortBy`, `sortOrder` and `after` for a reason?

Reply:
Seems good to me. And given that the whole project is open source, if we're trying to hide it then that isn't going to work for long 😂.

Reply:
Great points! Addressed these issues now @flea89 
Thanks